src/Mayaqua/Unix.c

```diff
@@ -1162,7 +1162,9 @@ bool UnixRunW(wchar_t *filename, wchar_t *arg, bool hide, bool wait)
 bool UnixRun(char *filename, char *arg, bool hide, bool wait)
 {
 	TOKEN_LIST *t;
+	char** args;

```

Comment:
The asterisk(s) should be attached to the variable name, to maintain consistency with the rest of the source code.

Reply:
Thank you, I have almost forgot this pull request :) 
Do you need my new pull request for it or do it yourself?  (Not very familiar with how to improve existing pull request on github ).

Reply:
Unfortunately I don't have write access to your branch, as I'm not part of the team.

You can add changes to the pull request by committing them in your `Run_deadlock` branch.

Don't worry about squashing the commits, it can be done when merging the pull request. :)

Reply:
I have found git commit -amend can help improve the pull request in place, so it is there now with your feedback. Please check it.

Thanks

Reply:
You're welcome, thank you for applying the changes!